File: Fundamentos da Programação de Computadores/3. Estrutura sequencial/Exercícios Resolvidos/Ex1.c
// Faça um programa que receba quatro números inteiros, calcule e mostre a soma desses números.

#include <stdlib.h>
#include <stdio.h>

int main()
{
    int Numero1, Numero2, Numero3, Numero4, soma = 0; // soma = 0 para n guardar lixo

    scanf("%d %d %d %d", &Numero1, &Numero2, &Numero3, &Numero4);

    soma = Numero1 + Numero2 + Numero3 + Numero4;

    printf("%d", soma);

    return 0;

} // end main

File: Java/Livros/Fundamentos da programação de computadores/Capítulo 3 Estrutura sequencial/Ex1.java

//Faça um programa que receba quatro números inteiros, calcule e mostre a soma desses números.
import java.util.Scanner;

public class Ex1 {
    public static void main(String args[]) {

        soma();
    }

    public static void soma() {

        // variaveis
        int[] N = new int[4];
        int soma = 0;

        // Objeto scarner
        Scanner dado = new Scanner(System.in);

        System.out.println("Digite 4 números inteiros: ");

        // loop principal
        for (int i = 0; i < N.length; i++) {
            N[i] = dado.nextInt();
            soma += N[i];
        }

        System.out.println("Soma: " + soma);

        dado.close();

    }

}
